plugins/jira-dashboard/package.json

```diff
@@ -30,6 +30,7 @@
     "@backstage/errors": "^1.2.3",
     "@backstage/plugin-catalog-react": "^1.9.3",
     "@backstage/theme": "^0.5.0",
+    "@material-ui/lab": "^4.0.0-alpha.61",
```

Comment:
Remove this since the "Alert" component is now included in mui5.

Reply:
updated